gammapy/background/models.py

```diff
@@ -71,3 +81,775 @@ def evaluate(self, x, y):
+        table['ENERG_HI'] = a_energy_hi
+        table['Bgd'] = a_bg
+
+        table.meta['E_THRES'] = a_energy_lo.flatten()[0].value
+
+        # name
+        table.meta['name'] = 'BACKGROUND'
+
+        return table
+
+    def to_fits_bin_table(self):
+        """Convert cube background model to binary table fits format.
+
+        Returns
+        -------
+        tbhdu : `~astropy.io.fits.BinTableHDU`
+            table containing the bg cube
+        """
+        # build astropy table
+        table = self.to_astropy_table()
+
+        # read name and drop it from the meta information, otherwise
+        # it would be stored as a header keyword in the BinTableHDU
+        name = table.meta.popitem('name')[1]
+
+        data = table.as_array()
+
+        header = fits.Header()
+        header.update(table.meta)
+
+        tbhdu = fits.BinTableHDU(data, header, name=name)
+
+        # Copy over column meta-data
+        for colname in table.colnames:
+            tbhdu.columns[colname].unit = str(table[colname].unit)
+
+        # TODO: this method works fine but the order of keywords in the table
+        # header is not logical: for instance, list of keywords with column
+        # units (TUNITi) is appended after the list of column keywords
+        # (TTYPEi, TFORMi), instead of in between.
+        # https://github.com/gammapy/gammapy/issues/298
+
+        return tbhdu
+
+    def to_fits_image(self):
+        """Convert cube background model to image fits format.
+
+        Returns
+        -------
+        hdu : `~astropy.io.fits.ImageHDU`
+            image containing the bg cube
+        """
+        hdu = fits.ImageHDU(data=self.background.value)
+        # TODO: store units (of bg) somewhere in header??!!!!
+        # TODO: implement WCS object to be able to read the det coords -> done
+        # TODO: energy binning: store in HDU table like for SpectralCube class
+
+        # get WCS object
+        wcs = self.det_wcs
+
+        # Now, write out the WCS object as a FITS header
+        wcs_header = wcs.to_header()
+
+        # transfering header values
+        # need to copy necessary values one by one
+        # TODO: can this be simplified? just copy the header doesn't
+        #       work, since it overwrites important keywords (for
+        #       instance the ones for the energy axis)
+        hdu.header['CTYPE1'] = wcs_header['CTYPE1']
+        hdu.header['CTYPE2'] = wcs_header['CTYPE2']
+        hdu.header['CUNIT1'] = wcs_header['CUNIT1']
+        hdu.header['CUNIT2'] = wcs_header['CUNIT2']
+        hdu.header['CDELT1'] = wcs_header['CDELT1']
+        hdu.header['CDELT2'] = wcs_header['CDELT2']
+        hdu.header['CRPIX1'] = wcs_header['CRPIX1']
+        hdu.header['CRPIX2'] = wcs_header['CRPIX2']
+        hdu.header['CRVAL1'] = wcs_header['CRVAL1']
+        hdu.header['CRVAL2'] = wcs_header['CRVAL2']
+
+        return hdu
+
+    def write(self, outfile, format='bin_table', write_kwargs=None):
```

Comment:
Just call the default format `table` instead of `bin_table`.
We'll never use FITS ascii tables.